kasa/smart/smartchilddevice.py

```diff
@@ -79,9 +83,21 @@ async def _update(self, update_children: bool = True):
         self._last_update_time = now
 
     @classmethod
-    async def create(cls, parent: SmartDevice, child_info, child_components):
+    async def create(
+        cls,
+        parent: SmartDevice,
+        child_info,
+        child_components,
+        protocol: SmartProtocol | None = None,
+        *,
+        last_update: dict | None = None,
```

Comment:
This feels somehow very wrong, can we not read this from the parent object?

Reply:
Which bit feels wrong? We don't want to provide this data unless the parent is of a different protocol. We can't derive it from the parent because the parent doesn't store `smart` last info data, it stores it's own `smartcamera` data.

Reply:
Oh, okay. Well, better to add a note to the docstring for that, and perhaps it could be renamed to make its usage clear? I'm out of ideas at the moment, though :-)

Reply:
Docstring updated